lang/cs/Org.Apache.REEF.Evaluator.Tests/ContextRuntimeTests.cs

```diff
@@ -37,13 +42,7 @@ public sealed class ContextRuntimeTests
         public void TestContextEvents()
         {
             const string hello = "Hello!";
-            var contextConfig = ContextConfiguration.ConfigurationModule
-                .Set(ContextConfiguration.Identifier, "ID")
-                .Set(ContextConfiguration.OnContextStart, GenericType<ContextEventHandler>.Class)
-                .Set(ContextConfiguration.OnContextStop, GenericType<ContextEventHandler>.Class)
-                .Set(ContextConfiguration.OnMessage, GenericType<ContextEventHandler>.Class)
-                .Build();
-            
+            var contextConfig = GetContextEventHandlerContextConfiguration();
             var injector = TangFactory.GetTang().NewInjector();
 
             var handler = new ContextEventHandler();
```

Comment:
This test is not to ensure that the handler is injected, but rather to ensure that events are invoked on the handler. We cannot simply use `injector.GetInstance` to make sure that the handler is invoked because internally, `ContextRuntime` forks off the `injector` we pass in prior to instantiated the handler. Thus, we will need to use `BindVolatileInstance` here for testing.